pkg/crc/api/handlers.go

```diff
@@ -68,7 +68,6 @@ func (h *Handler) Start(args json.RawMessage) string {
 		if err != nil {
 			startErr := &client.StartResult{
 				Success: false,
-				Name:    h.MachineClient.GetName(),
```

Comment:
We'll also need to update the structs on the trays..

Reply:
Yes I opened this one for macOS. https://github.com/code-ready/tray-macos/pull/108
For Windows, is there something needed ?